contracts/MultiSigWallet.sol

```diff
@@ -86,78 +94,60 @@ contract MultiSigWallet {
     /* Storage */
 
     /**
-      It denotes the total number of confirmations required for an
-      transaction to be executed.
+     * Specifies the number of confirmations required for a proposed
+     * transaction to be executed.
      */
-    uint8 public required;
-    /** It maps status and who approved that transactionId. */
-    mapping (bytes32 => Confirmation) public confirmations;
-    /** It helps to direct lookup whether an wallet is already present or not */
+    uint256 public required;
```

Comment:
Updating your comment about change from uint8 to uint256 for documentation purpose :-
"There were security flows because of that (overflow for example). We do not need this optimization in this stage."
Our reasoning was more than 256 owners of an wallet is highly unlikely.